cpp/src/plasma/store.cc

```diff
@@ -259,6 +259,46 @@ void PlasmaObject_init(PlasmaObject* object, ObjectTableEntry* entry) {
   object->device_num = entry->device_num;
 }
 
+void PlasmaStore::RemoveGetRequest(GetRequest* get_request) {
```

Comment:
This function is just factored out from `PlasmaStore::ReturnFromGet`. We shouldn't be using raw pointers.